tests/test_endpoint.py

```diff
@@ -23,7 +23,16 @@ def test_server_health_returns_200():
 def test_server_list_actions_returns_200():
     request, response = app.test_client.get("/actions")
     assert response.status == 200
-    assert len(response.json) == 3
+    assert len(response.json) == 4
+
+    # ENSURE TO UPDATE AS MORE ACTIONS ARE ADDED IN OTHER TESTS
+    expected = [
+        {"name": "custom_async_action"},
+        {"name": "custom_action"},
+        {"name": "custom_action_exception"},
```

Comment:
I'd leave a top level comment where these actions are defined (in `test_actions.py`) and a reference for the 4th mock validation action.

Reply:
what's a top level comment? 🙈

Reply:
Oh I just meant a comment in L29 for all the 3 actions + a 2nd comment above the mock validation action, that's what I meant by top 😄

Reply:
Got it! 😂